CMakeLists.txt

```diff
@@ -35,6 +35,12 @@ endif()
 # so we provide an option similar to BUILD_TESTING, but just for THM.
 option(THM_BUILD_TESTING "enable testing for thm" ON)
 
+# Add -O0 to remove optimizations when using gcc
```

Comment:
You don't need this, please pass `-DCMAKE_BUILD_TYPE=Debug` when compiling

Reply:
make "DCMAKE_BUILD_TYPE=Debug" ?

Reply:
`cmake -DCMAKE_BUILD_TYPE=Debug`